src/pca_report_library/templates/generate_template.py

```diff
@@ -79,11 +77,8 @@ def main():
         print("Copying Asset Folder for LaTeX report to current dir....")
 
     for i in os.listdir(temp_working_dir):
-        shutil.move(os.path.join(temp_working_dir, i), original_working_dir)
+        if not os.path.exists(os.path.join(temp_working_dir, i)):
+            shutil.move(os.path.join(temp_working_dir, i), original_working_dir)
```

Comment:
Mirroring above suggestion while also DRYing.

```suggestion
        temp_path = os.path.join(temp_working_dir, temp_entry)
        if not os.path.exists(temp_path):
            shutil.move(temp_path, original_working_dir)
```

Reply:
I have made the suggested changes here e69a69db6250f080e448fd6f8398bada51dc1e9c but also got rid of the "not" in the if statement because it was doing the reverse of what was needed.